algebird-core/src/main/scala/com/twitter/algebird/lsh/hashing/HashFamily.scala

```diff
@@ -0,0 +1,14 @@
+package com.twitter.algebird.lsh.hashing
+
+import com.twitter.algebird.lsh.vector.BaseLshVector
+
+trait Hasher {
```

Comment:
Actually there is a scala.util.hashing.Hashing: http://www.scala-lang.org/api/2.11.4/index.html#scala.util.hashing.Hashing

Can we use that trait?